server/src/modules/user_ensembles/dto/create-user-ensemble.dto.ts

```diff
@@ -2,9 +2,9 @@ import { IsNotEmpty } from 'class-validator';
 
 export class CreateUserEnseble {
   @IsNotEmpty()
-  user_id: string;
+  userId: string;
```

Comment:
100% this won't mess with the existing code?

Reply:
I changed it everywhere in the service where it was used that way and I tested to create an ensemble and it looks like it's working